components/atom/card/src/index.scss

```diff
@@ -0,0 +1,34 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$c-atom-card-highlight: #fffed4 !default;
```

Comment:
Isn't there a sui-theme color for that ? If not I think it should be defined...

Reply:
Can you please create an issue about this on `sui-theme` repo so I can create the proper PR on that repo?

Reply:
done!
https://github.com/SUI-Components/sui-theme/pull/160